### DISCUSSION TITLE
its 12:42 am don't ask ty

### DISCUSSION THREAD

**Discussion by IMGROOT2** [For-0/vocabustudy#181] [Announcements]
![forzero full logo](https://user-images.githubusercontent.com/116324098/197456021-1946540f-3290-4773-b20b-b6fdbd111182.png)
![smaller logo](https://user-images.githubusercontent.com/116324098/197456023-fb4594b8-f16d-4eb2-baf5-960f7a9360ff.png)

do i have any clue why it shows "date modified" as 5:43 am?

no.

i present to you: the unofficial forzero logos!

**Comment by IMGROOT2** [comment]
![image](https://user-images.githubusercontent.com/116324098/197460647-a6ce6b96-43b9-4f67-bac8-a72952e89ec5.png)

also its a wip and i'm gonna change the colors if it's okay, but what do you think? also, I want to put something in the top right but idk

**Comment by grimsteel** [comment]
> unofficial

Not anymore

I kind of like the old design better - I think the list of things looks better than a long line.
Also, people are not that likely to know what forzero applications is, so the name(s) of the people who made it would be better
